cli/tests/test_no_abandoned_test_files.rs

```diff
@@ -0,0 +1,24 @@
+use std::fs;
+use std::path::PathBuf;
+
+#[test]
+fn test_no_abandoned_test_files() {
+    let test_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("tests");
+    let runner_path = test_dir.join("runner.rs");
+    let runner = fs::read_to_string(&runner_path).unwrap();
+    let entries = fs::read_dir(test_dir).unwrap();
+    for entry in entries {
+        let path = entry.unwrap().path();
+        if let Some(ext) = path.extension() {
+            let name = path.file_stem().unwrap();
+            if ext == "rs" && name != "runner" {
+                let name = name.to_str().unwrap();
+                assert!(
+                    runner.contains(name),
```

Comment:
Nit: I'd search directly for `mod {name};` here, to match the error for example.